app/schema/api.py

```diff
@@ -154,6 +154,7 @@ class VlabAccountCreationIn(BaseModel):
 
     id: UUID
     name: str
+    balance: Decimal = D0
```

Comment:
We should enforce balance >=0, unless there is a use case to create an account with negative balance.

Reply:
I've added the validation and a test.